sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -97,6 +96,10 @@ namespace Azure { namespace Core {
         return DateTime(DateTime::time_point(DateTime::duration(dtRepresentation)));
       }
 
+      /** Creates a new ContextSharedState object with no deadline and no value.
```

Comment:
Historically we've added comments on the line after the `/**`. Is there a reason for this new style?

Reply:
I was having problems with doxygen finding the @brief description, this was an attempt at trying to fix that.

I'll move it to the more standard form.